docs/source/guide/explanation/additional_features/config_input_size.rst

```diff
@@ -0,0 +1,31 @@
+Configurable Input Size
+=======================
+
+You have the option to work with smaller input resolutions to expedite training and inference times,
```

Comment:
```suggestion
This feature make OTX use smaller input resolutions to expedite training and inference times,
```

I personally prefer to avoid using the first word as `You` at the document. I'm not sure what is the right way. So, please feel free to ignore this comment. Same as below contents for me.

Reply:
I updated it